tests/reg_tests/test_DVConstraints.py

```diff
@@ -163,7 +186,11 @@ def c172_test_twist(self, DVGeo, DVCon, handler):
         # change the DVs
         xDV = DVGeo.getValues()
         xDV["twist"] = np.linspace(0, 10, self.nTwist)
-        DVGeo.setDesignVars(xDV)
+        if self.child:
+            # Twist needs to be set on the parent FFD to get accurate derivatives
```

Comment:
I can confirm this is the same approach I use, but is this documented anywhere? What is the reason behind this?

Reply:
I don't think it's documented anywhere mostly because manually setting design variables is not too common. This is not specific to twist actually. My understanding is that `setDesignVars` should always be called on the parent, which will recursively set the DVs on all children. It just so happens that shape variables can be applied directly on a child FFD without causing issues. I can change the approach for all DVs to be consistent if you think that's clearer.